index.js

```diff
@@ -15,9 +15,11 @@ const defaults = require('./lib/defaults');
 const logger = require('./lib/logger');
 const helmet = require('helmet');
 
+const customConfig = {};
+
 const getConfig = function getConfig() {
   const args = [].slice.call(arguments);
-  return Object.assign.apply(null, [{}, defaults].concat(args));
+  return Object.assign.apply(null, [{}, defaults, customConfig].concat(args));
```

Comment:
ah of course it is, and yes, out of scope, this does seem to be needlessly complex though